src/pages/søknad/steg/formue/dinformue-nb.ts

```diff
@@ -30,5 +30,5 @@ export default {
     'harKontanter.beløp': 'Hvor mye penger har du?',
 
     'button.fjernRad': 'Fjern rad',
-    'button.leggTil': 'Legg til',
+    'button.leggTil': 'Legg til et annet kjøretøy',
```

Comment:
Tror vi skriver `Legg til annet kjøretøy` en eller fler andre stedet. Jeg synes vi skal gå for denne alle plassene.